node/crypto/wesolowski_frame_prover.go

```diff
@@ -579,15 +579,24 @@ func (w *WesolowskiFrameProver) VerifyWeakRecursiveProof(
 	}
 }
 
+func (w *WesolowskiFrameProver) CalculateChallengeProofDifficulty(
+	increment uint32,
+) uint32 {
+	difficulty := 200000 - (increment / 4)
```

Comment:
the increment check is needed because this is a uint32, it'll underflow

Reply:
Reordered the function to only do the calculation on increments less than 700k